python/cudf/io/csv.py

```diff
@@ -380,7 +393,7 @@ def read_csv(filepath_or_buffer, lineterminator='\n',
 
 
 def read_csv_strings(filepath_or_buffer, lineterminator='\n',
-                     quotechar='"', quoting=True, doublequote=True,
+                     quotechar='"', quoting='minimal', doublequote=True,
```

Comment:
Update default to 0 here as well
```suggestion
                     quotechar='"', quoting=0, doublequote=True,
```